android/uhabits-android/src/main/java/org/isoron/uhabits/receivers/WidgetReceiver.java

```diff
@@ -105,6 +104,9 @@ public void onReceive(final Context context, Intent intent)
                             data.getTimestamp());
                     break;
                 case ACTION_SET_NUMERICAL_VALUE:
+                    // close Notification drawer
```

Comment:
Same comment above about inline comments. In this case, please either remove the comment or move the line into a `closeNotificationDrawer()` method.